CONTRIBUTING.md

```diff
@@ -0,0 +1,270 @@
+state and "force push" it to your github branch. Github understands this action, and
+does sensible things in the online comment history. Do not pile patches on
+patches to rework your branch. Any relevant information from the github
+comments section should be re-worked into your patch set, as the ultimate place
+where your patches are documented is in the git log, and not in the github
+comments section.
+
+For more information on github "force push" workflows see
+[here](http://blog.adamspiers.org/2015/03/24/why-and-how-to-correctly-amend-github-pull-requests/).
+
+It is perfectly fine for your PR to contain more than one patch - use
+as many patches as you need to implement the Request (see the previously
+mentioned "small patch" thoughts). Each PR should only cover one
+topic - if you mix up different items in your patches or PR then you
+will most likely be asked to rework them.
+
+### Re-vendor PRs
+
+If you are raising a PR to update the vendored copy of one or more golang
+packages, there are two critical pieces of information you need to add to the
+commit body:
+
+- A brief explanation as to why this re-vendor is required.
+
+  For example, it might be that an important bug fix or new feature is
+  required. It might even be that a particular commit is needed. If so, please
+  state that.
+
+- The range of commits being added for these third-party packages.
+
+  It is possible that re-vendoring a particular package will also result in updates to
+  other dependent packages. However, it is important to include the commit range
+  (even if it is a big) for the primary package(s) the re-vendor PR has been
+  raised for.
+
+  These details allow for easier problem-determination in the future should
+  the re-vendor PR introduce bugs or behavioural changes.
+
+  Generate the list of new commits added to the primary re-vendored
+  package by comparing the previous and latest commits for the package being re-vendored.
+
+  For example, if a new version of `libcontainer` (part of the `runc`)
+  repository is required:
+
+  1. Determine the previous and latest commits for the package by looking at
+     the `diff` of the `Gopkg.toml` file in your branch.
+
+  1. Run the commands below:
+
+     ```
+     $ cd $GOPATH/src/github.com/opencontainers/runc
+     $ old_commit="..."
+     $ new_commit="..."
+     $ git log --no-merges --abbrev-commit --pretty=oneline "${old_commit}..${new_commit}") | sed 's/^/    /g'
+     ```
+
+  The output of the last command above should be pasted directly into the
+  commit "as-is". Note that the four space indent added by the `sed` command is
+  used to force github to render the list in a fixed-width font which makes the
+  list easier to read.
+
+## Patch format
+
+Beside the `Signed-off-by` footer, we expect each patch to comply with the
+following format:
+
+```
+Subsystem: Change summary (no longer than 75 characters)
+
+More detailed explanation of your changes: Why and how.
+Wrap it to 72 characters.
```

Comment:
Could we please remove that particular check?

Reply:
Oh good catch (see https://github.com/kata-containers/agent/pull/77#discussion_r158183877). Removing...